tests/ApiCore/Tests/Unit/GapicClientTraitTest.php

```diff
@@ -377,17 +379,27 @@ public function testBuildClientOptions($options, $expectedUpdatedOptions)
 
     public function buildClientOptionsProvider()
     {
+        $apiConfig = new ApiConfig();
+        $apiConfig->mergeFromJsonString(
+            file_get_contents(__DIR__.'/testdata/test_service_grpc_config.json'));
```

Comment:
Could we please put the closing parenthesis on a newline?

```php
$apiConfig->mergeFromJsonString(
    file_get_contents(__DIR__.'/testdata/test_service_grpc_config.json')
);
```

Reply:
done. Habits for Python :)